doc/advanced-configuration/outgoing-connections.md

```diff
@@ -450,3 +451,44 @@ Run the following function in the MongooseIM shell to verify that the connection
 ```
 
 Note that the output might differ based on your ElasticSearch cluster configuration.
+
+## RabbitMQ connection setup
+
+RabbitMQ backend for [`mod_event_pusher`](../modules/mod_event_pusher.md)
+requires a `rabbit` pool defined in the `outgoing_pools` option.
+They can be defined as follows:
+
+```erlang
+{ougtoing_pools, [
+ {rabbit, host, Tag, WorkersOptions, ConnectionOptions}
+]}.
+```
+
+Notice that `Host` parameter is set to atom `host`. This basically means that
+MongooseIM will start as many `rabbit` pools as XMPP hosts are served by
+the server.
+
+The `Tag` parameter must be set to `event_pusher` in order to be able to use
+the pool for [`mod_event_pusher_rabbit`](../modules/mod_event_pusher_rabbit.md).
+Any other `Tag` can be used for any other RabbitMQ connection pool.
+
+The `ConnectionOptions` list can take following parametrs as `{key, value`} pairs:
+
+* **amqp_host** (default: `"localhost"`) - Defines RabbitMQ server host (domain or IP address);
+* **amqp_port** (default: `5672`) - Defines RabbitMQ server AMQP port;
+* **amqp_username** (default: `<<"guest">>`) - Defines RabbitMQ server username;
+* **amqp_password** (default: `<<"guest">>`) - Defines RabbitMQ server password;
+* **confirms_enabled** (default: `false`) - Enables/disables one-to-one publishers confirms;
+
+### Example
+
+```erlang
+{ougtoing_pools, [
+ {rabbit, host, event_pusher, [{workers, 20}],
+  [{amqp_host, "localhost"},
+   {amqp_port, 5672},
+   {amqp_username, <<"guest">>},
```

Comment:
I would rather do not use binaries in configs. Using strings for amqp_username and amqp_password  would be more consistent.

Reply:
Good point! However, I can only change that for the connection parameters. Changing this for exchanges would end up with calling `binary:list_to_bin/1` for every "event" coming in.